irc.html

```diff
@@ -13,21 +13,21 @@ <h1>Channels</h1>
   </p>
   <ul>
     <li>
-      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#Minetest</a></strong>
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel where everyone is welcome to discuss the game or request support.
     </li>
+    
     <li>
-      <strong>
-        <a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a>
-      </strong>
-      (<a href="http://irc.minetest.net/minetest-dev">logs</a>):
-      development channel, only serious stuff; note that webchat is muted due to too much spam/flood.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-dev">#Minetest-Dev</a></strong>
+        (<a href="http://irc.minetest.net/minetest-dev">Logs</a>):
+        Development channel, strictly engine or minetest_game development talk only; note that webchat is muted due to too much spam/flood.
```

Comment:
Line too long in code, i assume it wraps when displayed onthe webpage.

Reply:
A web browser will indeed auto-wrap, so I wouldn't worry much about long line.

Reply:
I mean, long line is best avoided in the code, if possible.

Reply:
@paramat Better? And at least least nothing changed visually on Safari, just the editing in a console window gets easier with the line break indeed.